src/lib/services/types/nft.ts

```diff
@@ -28,11 +28,11 @@ export const zNft = z
     uri: val.uri,
     tokenId: val.token_id,
     description: val.description,
-    isBurned: val.is_burned,
+    isBurned: val.is_burned || null,
```

Comment:
I think this must be ?? not || because if val.is_burned is false, the value will be null (I pretty sure we don't want this result).